hopper/flash_api.cpp

```diff
@@ -17,44 +16,25 @@
 #include "heuristics.h"
 #include "cuda_check.h"
 
-// Copied from https://github.com/pytorch/pytorch/commit/7931eee5c5ebcdf468bff4d308510b03355cd909
-// This is so that we can pass in torch.dtype as a parameter to the function.
-#if TORCH_VERSION_MAJOR < 2 || (TORCH_VERSION_MAJOR == 2 && TORCH_VERSION_MINOR < 4)
```

Comment:
Note: I had to remove this because we're no longer using PyBind. This will probably break with PyTorch < 2.4, is this fine, or do we want a backup solution for older PyTorch versions?

Reply:
that's fine, we won't support pytorhc < 2.4